src/runtime/components/navigation/Pagination.vue

```diff
@@ -79,13 +97,29 @@ export default defineComponent({
       type: Object as PropType<Partial<Button>>,
       default: () => appConfig.ui.pagination.default.inactiveButton
     },
+    prevIconClass: {
```

Comment:
I'd rather have a `prevButton` and a `nextButton` props that are a `Partial<Button>` instead of 6 props.